### ISSUE TITLE
Database Setup Docs potential bug - incorrect for Postgres 15?

### ISSUE THREAD

**Issue by rifen** [nautobot/nautobot#3927]
### Environment
* Nautobot version (Docker tag too if applicable): 1.5.20
* Python version: pyenv(python3.7.16)
* Database platform, version: Postgres 15.2/Redis
* Middleware(s):

### Steps to Reproduce
1. Follow docs to install 1.5.20
2. Run nautobot-server migrate for the first time
3. 

### Expected Behavior
Nautobot populates the postgres DB

### Observed Behavior
```
  File "/opt/nautobot/.pyenv/versions/3.7.16/lib/python3.7/site-packages/django/db/migrations/executor.py", line 91, in migrate
    self.recorder.ensure_schema()
  File "/opt/nautobot/.pyenv/versions/3.7.16/lib/python3.7/site-packages/django/db/migrations/recorder.py", line 70, in ensure_schema
    raise MigrationSchemaMissing("Unable to create the django_migrations table (%s)" % exc)
django.db.migrations.exceptions.MigrationSchemaMissing: Unable to create the django_migrations table (permission denied for schema public
LINE 1: CREATE TABLE "django_migrations"
```
Related:
https://networktocode.slack.com/archives/C01NWPK6WHL/p1686270343745979
https://stackoverflow.com/questions/74110708/postgres-15-permission-denied-for-schema-public

Fix to be added to Database Setup docs potentially: `ALTER DATABASE nautobot OWNER TO nautobot;`

**Comment by gsnider2195** [comment]
Verified on a local install of postgres 15

**Comment by gsnider2195** [comment]
Fixed by #4730
